### PR TITLE
Feature/accounts crud - Added test cases for all four CRUD operations for Accounts

### PR DESCRIPTION
**Pull request by dip06ece**
Added all four CRUD operations tests and tested and found okay.

### CODE REVIEW

**Comment by dip06ece** [comment]
Please, review the test case.

File: src/test/java/com/tg5/integration/controller/AccountsControllerTest.java [hunk 1]
```diff
@@ -2,22 +2,71 @@
...
 import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.equalTo;
 
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
 public class AccountsControllerTest extends BaseTest {
+    private static Integer createdAccountId;
 
     @Test
-    public void test() {
+    @Order(1)
```
Comment (vytaux) [inline src/test/java/com/tg5/integration/controller/AccountsControllerTest.java]: Can you tell me about the order? Why is this required? I remember prof saying tests are supposed to be independent. And having an instance variable createdAccountId means you are creating a test sequence. You should set up your data before test and then test it without relying on previous tests.

**Review by vytaux — CHANGES_REQUESTED** [review]

File: src/test/java/com/tg5/integration/controller/AccountsControllerTest.java (hunk 1 above)
Comment (dip06ece) [reply src/test/java/com/tg5/integration/controller/AccountsControllerTest.java]: Create test -- Create one account 
Delete/ update test - work on that account 
Otherwise how can they know which account to work on!

Comment (dip06ece) [reply src/test/java/com/tg5/integration/controller/AccountsControllerTest.java]: After the test database should not have any effect. That's why order is defined in a way Create, read, update, delete.
